### ISSUE TITLE
PO: Unable to select existing annotation via keyboard

### ISSUE THREAD

**Issue by asimpson** [grafana/grafana#56564]
Note the outline skipping over the name of the annotation.
![annotation-keyboard](https://user-images.githubusercontent.com/1048831/194582677-4fc57ead-49ac-434d-9494-14f898843d7f.gif)
